tools/sidekick/config.json

```diff
@@ -0,0 +1,12 @@
+{
+  "project": "adobe-io",
```

Comment:
add `"host":  {your production host}"

as per https://www.hlx.live/docs/sidekick#customizing-the-sidekick